.github/workflows/SDK-Suppressions-Label.yaml

```diff
@@ -24,17 +24,14 @@ jobs:
       - name: Setup Node and install deps
         uses: ./.github/actions/setup-node-install-deps
 
-      - name: Get GitHub PullRequest Changed Files
-        shell: pwsh
+      - name: Get GitHub PullRequest Changed Suppression Files
```

Comment:
Does this still need to be a separate step?  Or can the two github-script steps be combined into a single step?  Should `eng/tools/sdk-suppressions` directly call `getChangedFiles()`?